src/app/shared/component/abstract.service.ts

```diff
@@ -41,4 +43,14 @@ export abstract class AbstractService<T extends AbstractModel> {
 	public update(item: T): Observable<T> {
 		return this.httpService.put(`/${this.baseUrl}/${item._id}`, item);
 	}
+
+	public getPaginated(skip: number, limit: number, sortBy: string, sortOrder: number): Observable<Array<T>> {
+		const params = new HttpParams()
+			.set('skip', skip.toString())
+			.set('limit', limit.toString())
+			.set('sortBy', sortBy)
+			.set('sortOrder', sortOrder.toString());
```

Comment:
make sure you don't make any of the sort options mandatory. what if i want to use the pagination without sorting options, can i ?